setup.py

```diff
@@ -8,7 +8,7 @@
 setup(
   name = 'panml', # package name     
   packages = find_packages(exclude=['test']), # package name
-  version = '0.0.25', # version
+  version = '0.0.26', # version
```

Comment:
This one is a breaking change. I recommend incrementing the major version rather than the patch version